amqtt/mqtt/protocol/handler.py

```diff
@@ -552,11 +552,6 @@ async def _send_packet(self, packet):
             )
         except (ConnectionResetError, BrokenPipeError):
             await self.handle_connection_closed()
-        except asyncio.CancelledError:
-            raise
-        except BaseException as e:
-            self.logger.warning("Unhandled exception: %s" % e)
```

Comment:
Also here no additional context is added and it's fairly clear that the exception wasn't handled.

Reply:
I restored the exception handling and only replaced `BaseException` with `Exception` here. I think I need to have a much deeper look into the code and more testing before changing the exception handling.